src/main/java/uk/gov/hmcts/reform/sscs/service/SscsDocumentService.java

```diff
@@ -50,4 +74,69 @@ public void filterByDocTypeAndApplyAction(List<SscsDocument> sscsDocument, Docum
             .forEach(action);
     }
 
+    public List<PdfDocument> sizeNormalisePdfs(List<PdfDocument> pdfDocuments) {
+
+        List<PdfDocument> normalisedPdfs = new ArrayList<>();
+
+        for (PdfDocument pdfDoc : pdfDocuments) {
+
+            AbstractDocument updatedSscsDocument;
+            Pdf updatedPdf;
+
+            Optional<Pdf> resized = resizedPdf(pdfDoc.getPdf());
+
+            if (resized.isPresent()) {
+                updatedPdf = resized.get();
+                updatedSscsDocument = saveAndUpdateDocument(updatedPdf, pdfDoc.getDocument());
+            } else {
+                updatedPdf = pdfDoc.getPdf();
+                updatedSscsDocument = pdfDoc.getDocument();
+            }
+            normalisedPdfs.add(PdfDocument.builder().document(updatedSscsDocument).pdf(updatedPdf).build());
+        }
+        return normalisedPdfs;
+    }
+
+    public AbstractDocument saveAndUpdateDocument(Pdf pdf, AbstractDocument document) {
+
+        String pdfFileName = document.getValue().getDocumentFileName() + ".pdf";
+
+        ByteArrayMultipartFile file = ByteArrayMultipartFile
+            .builder()
+            .content(pdf.getContent())
+            .name(pdfFileName)
+            .contentType(APPLICATION_PDF)
+            .build();
+
+        log.info("About to upload resized document [" + pdfFileName + "]");
+
+        try {
+            UploadResponse upload = evidenceManagementService.upload(singletonList(file), "sscs");
+            String location = upload.getEmbedded().getDocuments().get(0).links.self.href;
+            DocumentLink documentLink = DocumentLink.builder().documentUrl(location).build();
+            document.getValue().setResizedDocumentLink(documentLink);
+
+        } catch (Exception e) {
+            log.error("Failed to store resized pdf document but carrying on [" + pdfFileName + "]", e);
+        }
+        return document;
+    }
+
+    public Optional<Pdf> resizedPdf(Pdf originalPdf) throws BulkPrintException {
+
+        try {
+            PDDocument document = PDDocument.load(originalPdf.getContent());
```

Comment:
Better to put the try within the PDocument.load() so that we don't forget to close the document.

i.e. 
try (PDDocument document = PDDocument.load(originalPdf.getContent())) {
...
}